pkg/settings/setting.go

```diff
@@ -24,6 +24,7 @@ var (
 	EngineNewestVersion             = NewSetting("engine-newest-version", "v17.12.0")
 	EngineSupportedRange            = NewSetting("engine-supported-range", "~v1.11.2 || ~v1.12.0 || ~v1.13.0 || ~v17.03.0 || ~v17.06.0 || ~v17.09.0 || ~v18.06.0 || ~v18.09.0")
 	FirstLogin                      = NewSetting("first-login", "true")
+	GlobalRegistryEnabled           = NewSetting("global-registry-enabled", "false")
```

Comment:
Who sets this setting to true?
How is it used

Reply:
UI will use it. UI need to know if global registry is enabled to decide if we should global registry is an option in adding registry credential page